apps/omg_api/lib/root_chain_coordinator/core.ex

```diff
@@ -105,7 +104,11 @@ defmodule OMG.API.RootChainCoordinator.Core do
          root_chain_height,
          new_reported_sync_height
        ) do
-    current_synced_height <= new_reported_sync_height and new_reported_sync_height <= root_chain_height
+    _ =
+      if new_reported_sync_height > root_chain_height,
+        do: Logger.warn("Eth height back to #{inspect(root_chain_height)}, had #{inspect(new_reported_sync_height)}")
+
+    current_synced_height <= new_reported_sync_height
```

Comment:
this inequality also is unfortunately incorrect  :sob: 
second cycle of request_sync after reorg

Reply:
can you elaborate on that? reorgs (`root_chain_height` going backwards) don't seem to affect the synced height

Reply:
ok I get it. Client calls `RootChainCoordinator.get_synced_info`, which just got its `root_chain_height` back-tracked and it will process on an earlier height. But the problem lies not in this equality (this equality ensures that the client doesn't backtrack, which is v. important because it would otherwise process some events twice.

Problem lies in that the RootChainCoordinator can serve a backtracked height. If it is allowed to only ever progress, it seems it'll be ok. So `RootChainCoordinator.update_root_chain_height`. I'll make this amendment

Reply:
ah, I didn't see your comment on time but I applied a similar fix, thx!